onnxruntime/core/optimizer/initializer.h

```diff
@@ -141,7 +144,8 @@ class Initializer final {
       tensor_proto.set_raw_data(raw_data_.data(), raw_data_.size());
     } else {
       switch (data_type_) {
-        case ONNX_NAMESPACE::TensorProto_DataType_FLOAT16: {
+        case ONNX_NAMESPACE::TensorProto_DataType_FLOAT16: 
+        case ONNX_NAMESPACE::TensorProto_DataType_BFLOAT16: {
           tensor_proto.clear_int32_data();
           for (int i = 0; i < size_; i++) {
             tensor_proto.add_int32_data(float16_data_[i]);
```

Comment:
Should we first add a doc comment here that BFLOAT16 are als in int_32?
[https://github.com/onnx/onnx/blob/master/onnx/onnx-ml.proto#L517](https://github.com/onnx/onnx/blob/master/onnx/onnx-ml.proto#L517)